packages/opentelemetry-plugin-user-interaction/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "@opentelemetry/plugin-user-interaction",
+  "version": "0.4.0",
```

Comment:
Why not current version `0.3.2`?

Reply:
my understanding is that when it has the same version as current and it gets merged people are confused thinking it should be already available

Reply:
I think the `private: true` property is for this purpose. I wonder if we should have a parallel branch for changes that shouldn't get released in a patch release. named tracers would have been a good candidate too